server/docker-dev.env

```diff
@@ -18,7 +18,9 @@ DOMAIN_OVERRIDE=localhost:5000
 # Options: prod, preprod, dev
 MATH_ENV=dev
 PORT=5000
+
 SHOULD_USE_TRANSLATION_API=false
+GOOGLE_CREDENTIALS_BASE64=xxxxxxxxxxxxxxxxxxxxxxx...
```

Comment:
Don't set either of these by default, as setting `GOOGLE_CREDENTIALS_BASE64` forces codepath away from using `GOOGLE_CREDS_STRINGIFIED`, which could be confusing.

```suggestion
# GOOGLE_CREDENTIALS_BASE64=xxxxxxxxxxxxxxxxxxxxxxx...
# GOOGLE_CREDS_STRINGIFIED=xxxxxxxxxxxxxxxxxxxxxxx...
```